[2022-09-03T09:00:15.325+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: weather_data.check_weather manual__2022-09-03T09:00:12.135003+00:00 [queued]>
[2022-09-03T09:00:15.334+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: weather_data.check_weather manual__2022-09-03T09:00:12.135003+00:00 [queued]>
[2022-09-03T09:00:15.335+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2022-09-03T09:00:15.337+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 1
[2022-09-03T09:00:15.338+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2022-09-03T09:00:15.351+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): check_weather> on 2022-09-03 09:00:12.135003+00:00
[2022-09-03T09:00:15.357+0000] {standard_task_runner.py:52} INFO - Started process 2404 to run task
[2022-09-03T09:00:15.361+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'weather_data', 'check_weather', 'manual__2022-09-03T09:00:12.135003+00:00', '--job-id', '38', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpw6_72jve', '--error-file', '/tmp/tmpxj0gcj9p']
[2022-09-03T09:00:15.364+0000] {standard_task_runner.py:80} INFO - Job 38: Subtask check_weather
[2022-09-03T09:00:15.431+0000] {task_command.py:371} INFO - Running <TaskInstance: weather_data.check_weather manual__2022-09-03T09:00:12.135003+00:00 [running]> on host 3d56073112d8
[2022-09-03T09:00:15.475+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=weather_data
AIRFLOW_CTX_TASK_ID=check_weather
AIRFLOW_CTX_EXECUTION_DATE=2022-09-03T09:00:12.135003+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-09-03T09:00:12.135003+00:00
[2022-09-03T09:00:15.484+0000] {python.py:173} INFO - Done. Returned value was: {'coord': {'lon': 30, 'lat': 60}, 'weather': [{'id': 804, 'main': 'Clouds', 'description': 'пасмурно', 'icon': '04d'}], 'base': 'stations', 'main': {'temp': 11.1, 'feels_like': 9.94, 'temp_min': 10.72, 'temp_max': 11.81, 'pressure': 1019, 'humidity': 64, 'sea_level': 1019, 'grnd_level': 1019}, 'visibility': 10000, 'wind': {'speed': 5.34, 'deg': 288, 'gust': 6.73}, 'clouds': {'all': 100}, 'dt': 1662195230, 'sys': {'type': 2, 'id': 197864, 'country': 'RU', 'sunrise': 1662173958, 'sunset': 1662224402}, 'timezone': 10800, 'id': 534841, 'name': 'Лисий Нос', 'cod': 200}
[2022-09-03T09:00:15.500+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=weather_data, task_id=check_weather, execution_date=20220903T090012, start_date=20220903T090015, end_date=20220903T090015
[2022-09-03T09:00:15.541+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-09-03T09:00:15.561+0000] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
